crates/anonymization/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cloudproof_anonymization"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
It seems there is no change in this subcrate.

Reply:
the underlying version of crypto_core is still 0.7.x